evalme/classification.py

```diff
@@ -143,7 +144,7 @@ def naive(x, y, per_label=False, **kwargs):
                     # No-label if exception occurs
                     except Exception as e:
                         logger.error("Can't assign result for label.", exc_info=True)
-                        label = 'No-label'
+                        continue
```

Comment:
this error is misleading - should be information provided, or there is no error here, hence no exception needed

Reply:
Added type info in error text

Reply:
I can't understand how the exception can happen due to an unexistent `t`,
and why we can't rely on simply retrieving `t` and checking for `labels is None` - without explicit `.get('hypertextlabels')` ? It's very confusing

Reply:
> I can't understand how the exception can happen due to an unexistent t

Deleted exception handling on label extraction

> and why we can't rely on simply retrieving t and checking for labels is None - without explicit .get('hypertextlabels') ? It's very confusing

html labels results can contain both hypertextlabels and htmllabels results, so we need to extract either of them